eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgNetworkDb.scala

```diff
@@ -182,8 +181,7 @@ class PgNetworkDb(implicit ds: DataSource) extends NetworkDb with Logging {
     inTransaction { pg =>
       using(pg.prepareStatement("SELECT short_channel_id from pruned WHERE short_channel_id=?")) { statement =>
         statement.setLong(1, shortChannelId.toLong)
-        val rs = statement.executeQuery()
-        rs.next()
+        statement.executeQuery().next()
```

Comment:
Not sure this behaves the same as before, since it's `hasNext` that actually calls `rs.next()` under the hood...

Reply:
Yeah, I wasn't sure about this one either, but there is a unit test so I guess it works. How about:

```scala
statement.executeQuery().headOption.nonEmpty
```

Reply:
Fixed with 305d0914d.